Apps/Database/src/Assets/docs/EmailAccountClosed.html

```diff
@@ -21,7 +21,7 @@
                     scope="col"
                     style="text-align: left; width:350px;"
                 >
-                    <div role="img" aria-label="Health Gateway Logo">
+                    <div aria-label="Health Gateway Logo">
```

Comment:
Changes to these .html files will show up on the next DB migration; do we want to create a migration now just to deal with them?

Reply:
Migration created.